src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -1524,7 +1524,11 @@ public SSTableReader getMaxSizeFile(Iterable<SSTableReader> sstables)
     {
         // skip snapshot creation during scrub, SEE JIRA 5891
         if(!disableSnapshot)
-            snapshotWithoutFlush("pre-scrub-" + System.currentTimeMillis());
+        {
+            long timestamp = System.currentTimeMillis();
+            String snapshotName = "pre-scrub-" + timestamp;
+            snapshotWithoutFlush(snapshotName, null, false, null, null, timestamp);
```

Comment:
I think now we can re-use the newly created method:
```
snapshotWithoutFlush(snapshotName, timestamp);
```

Reply:
good catch, thanks